source/tutorials_installation.md

```diff
@@ -0,0 +1,396 @@
+# ODAHU installation from scratch
```

Comment:
ODAHU quickstart?

Reply:
I would like to rename the `MLFlow` section to `Quickstart`. Because, quickstart means the most straightforward way to start using the product, not install it.
And I see no reason to add the `Odahu` prefix.

 "Installation from scratch" ?

Reply:
I think it’s better then to leave it like "Installation" and "Quickstart" for MLFlow section.

Reply:
Done in 825ffa427aacb6fea93ba809de132546b3eb64b4